.github/workflows/rolling_batch_integration.yml

```diff
@@ -85,6 +85,15 @@ jobs:
           serve -m test=file:/opt/ml/model/test/
           python3 rb_client.py scheduler_single_gpu bloom-560m
           docker rm -f $(docker ps -aq)
+      - name: Test llama2-7b-chat-gptq
+        working-directory: tests/integration
+        run: |
+          rm -rf models
+          python3 llm/prepare.py rolling_batch_scheduler llama2-7b-chat-gptq
```

Comment:
will this test work if we didn't install `autogptq` and `optimum` ?

Reply:
You can refer to this: https://github.com/deepjavalibrary/djl-serving/blob/master/tests/integration/llm/prepare.py#L649

Reply:
you can pass with requirements.txt to install those dependencies on the fly